xmtp_mls/src/groups/mod.rs

```diff
@@ -253,7 +253,10 @@ impl MlsGroup {
 
     // Load the stored MLS group from the OpenMLS provider's keystore
     #[tracing::instrument(level = "trace", skip_all)]
-    fn load_mls_group(&self, provider: impl OpenMlsProvider) -> Result<OpenMlsGroup, GroupError> {
+    pub fn load_mls_group(
```

Comment:
Doesn't seem like a big deal to make these all public but makes me wonder if this function should live inside `xmtp_mls/src/groups` instead?